### DISCUSSION TITLE
Release 5.0.0 - Add central panel configuration

### DISCUSSION THREAD

**Discussion by jmcollin78** [jmcollin78/versatile_thermostat#277] [Announcements]
# Content

This release contains the major change: add a central configuration panel [Central configuration panel](https://github.com/jmcollin78/versatile_thermostat/issues/239).

---
![New](https://github.com/jmcollin78/versatile_thermostat/blob/main/images/new-icon.png?raw=true)
# Major changes in version 5.0

You can now define a central configuration which will allow you to share certain attributes on all your VTherms (or only part of them). To use this possibility, you must:

1. Create a VTherm of type “Central Configuration”,
2. Enter the attributes of this central configuration
3. To use it in the other VTherms, you must reconfigure them and whenever possible check the "Use central configuration" box. This check box appears in all attribute groups that can use central configuration: main attributes, TPI, openings, movement, power, presence and advanced parameters.

The configurable attributes in the central configuration are listed here: [Parameter summary](https://github.com/jmcollin78/versatile_thermostat/blob/main/README.md#parameter-synthesis).

When changing the central configuration, all VTherms will be reloaded to take these changes into account.

Consequently, the entire configuration phase of a VTherm has been profoundly modified to be able to use the central configuration or overload the values of the central configuration with values specific to the VTherm being configured.

Note: the VTherm configuration screenshots have not been updated.

<hr /><em>This discussion was created from the release <a href='https://github.com/jmcollin78/versatile_thermostat/releases/tag/5.0.0'>Release 5.0.0 - Add central panel configuration</a>.</em>

**Comment by HACS-bank** [comment]
Central configuration is great.

Would it be possible to allow more than one "central config"?

I have two kinds of rooms with stats: UFH with zone valves, which are very slow to react, and electric radiators with individual switches with power monitoring and fast reaction. 

Two different 'standard configuration' entries would cover this ... it would need:

Named configurations when creating. 

Chooser to pick one of them.

Etc.
